sdk/cosmosdb/cosmos/src/utils/logger.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+* Cosmos Diagnostics used for all clients within the Cosmos package
+*/
+const cosmosDiagnostics = require('@azure/logger');
+export const cosmosDiagnosticsLogger = cosmosDiagnostics.createClientLogger("Cosmos Diagnostics");
+
+/**
+ * The log levels supported by the Cosmos Diagnostics.
+ * The log levels are:
+ * - verbose
+ */
+export const cosmosDiagnosticsToString = String(cosmosDiagnosticsLogger.verbose.bind(cosmosDiagnostics.log));
+
+export function setCosmosDiagnosticsLogLevel(){
+  cosmosDiagnosticsLogger.verbose.bind(cosmosDiagnostics.log)
+  cosmosDiagnostics.setLogLevel("verbose");
+  cosmosDiagnostics.log = (...args: any) => { console.log(...args); };
```

Comment:
The normal practice is to allow the customer to control these directly, rather than us inferring the appropriate log level to enable.

Also, the default browser behavior is to log to the appropriate level, (e.g. verbose is 'debug'), so that dev tool filters work as expected: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/core/logger/src/log.browser.ts

Versus node defaults to writing to stderr, which seems reasonable? https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/core/logger/src/log.ts

Can you help me understand your customer scenario here and why the default behaviors don't work for this SDK?

Reply:
This is a customer ask feature.  You can read more here. 
https://docs.microsoft.com/en-us/azure/cosmos-db/cosmosdb-monitor-resource-logs#:~:text=You%20can%20turn%20on%20diagnostic%20setting%20for%20Azure,specify%20which%20category%20of%20logs%20to%20collect.%20